根目录为工作目录时，fit启动失败

### 问题摘要 / Issue Summary

根据文档install之后，没有切换目录，在根目录启动失败

### 版本信息 / Version Information

其他 (请在下方说明) / Other (please specify below)

### 浏览器 / Browser

_No response_

### 操作系统 / Operating System

Windows 10

### 发生了什么？/ What happened?

start之后日志报错，同时在用户目录下tmp目录复制了jar包

### 期望的行为 / Expected Behavior

能够顺利启动

### 重现步骤 / Steps to Reproduce

cd fit-framework
mvn clean install
/build/bin/fit start

### 相关日志 / Relevant Logs

```shell
PS E:\home\fred\git\开源框架\fit-framework> build/bin/fit start
Running command: java -D"sun.io.useCanonCaches=true" -D"plugin.fit.dynamic.plugin.directory=/e/home/fred/git/开源框架/fit-framework" -jar fit-discrete-launcher-3.5.0-M6.jar
[2025-08-03 22:55:16.188] [INFO ] [main] [modelengine.fitframework.runtime.discrete.DiscreteFitRuntime] Prepare to start FIT application... [version=3.5.0-M6]
[2025-08-03 22:55:21.091] [INFO ] [main] [modelengine.fel.tool.support.DefaultToolFactoryRepository] Register factory[type=FIT] success.
[2025-08-03 22:55:21.096] [INFO ] [main] [modelengine.fel.tool.support.DefaultToolFactoryRepository] Register factory[type=HTTP] success.
[2025-08-03 22:55:21.531] [INFO ] [main] [modelengine.fit.http.server.handler.ReflectibleHttpHandlerRegistry] Register http handler group successfully. [group=modelengine.fit.actuator.ActuatorController]
[2025-08-03 22:55:21.811] [INFO ] [main] [modelengine.fit.http.server.handler.ReflectibleHttpHandlerRegistry] Register http handler group successfully. [group=modelengine.fit.http.server.handler.OptionsHttpHandler]
[2025-08-03 22:55:21.812] [INFO ] [main] [modelengine.fit.http.server.handler.ReflectibleHttpHandlerRegistry] Register http handler group successfully. [group=modelengine.fit.http.server.handler.StaticResourceHttpHandler]
[2025-08-03 22:55:21.911] [INFO ] [main] [modelengine.fit.http.server.handler.ReflectibleHttpHandlerRegistry] Register http handler group successfully. [group=modelengine.fit.http.openapi3.swagger.DocumentController]
[2025-08-03 22:55:22.017] [INFO ] [main] [modelengine.fit.http.server.handler.ReflectibleHttpHandlerRegistry] Register http handler group successfully. [group=modelengine.fit.log.LoggerController]
[2025-08-03 22:55:22.275] [INFO ] [main] [modelengine.fit.http.server.handler.ReflectibleHttpHandlerRegistry] Register http handler group successfully. [group=modelengine.fit.service.CheckController]
[2025-08-03 22:55:22.344] [INFO ] [main] [modelengine.fit.http.server.handler.ReflectibleHttpHandlerRegistry] Register http handler group successfully. [group=modelengine.fit.service.RegistryListener]
[2025-08-03 22:55:23.647] [INFO ] [main] [modelengine.fit.http.server.handler.ReflectibleHttpHandlerRegistry] Register http handler group successfully. [group=modelengine.fel.tool.mcp.server.McpServerController]
[2025-08-03 22:55:24.235] [INFO ] [main] [modelengine.fitframework.runtime.discrete.DiscreteFitRuntime] FIT application started. [version=3.5.0-M6]
[2025-08-03 22:55:24.270] [INFO ] [directory-monitor-fit-framework-thread-0] [modelengine.fit.plugin.dynamic.directory.DynamicPluginScanner] Dynamic plugin detected. [plugin=E:\home\fred\git\开源框架\fit-framework\build\fit-discrete-launcher-3.5.0-M6.jar]
[2025-08-03 22:55:24.296] [INFO ] [registry-client-thread-0] [modelengine.fit.service.RegistryClient] Register fitables successfully. All fitable services are ready.
[2025-08-03 22:55:24.307] [ERROR] [directory-monitor-fit-framework-thread-0] [modelengine.fit.plugin.dynamic.directory.DynamicPluginScanner] Failed to scan directory.
java.lang.IllegalStateException: Missing manifest in plugin JAR. [location=file:/C:/Users/geex38/AppData/Local/Temp/application-5997341732875804306/fit-discrete-launcher-3.5.0-M6-1754748338878048553.jar, entry=FIT-INF/plugin.xml]
        at modelengine.fitframework.plugin.support.JarPluginResolver.readPluginManifest(JarPluginResolver.java:115)
        at modelengine.fitframework.plugin.support.JarPluginResolver.loadMetadata(JarPluginResolver.java:63)
        at modelengine.fitframework.plugin.support.JarPluginResolver.resolve(JarPluginResolver.java:57)
        at modelengine.fitframework.plugin.support.DefaultPluginCollection.lambda$add$0(DefaultPluginCollection.java:69)
        at modelengine.fitframework.util.LockUtils.synchronize(LockUtils.java:138)
        at modelengine.fitframework.plugin.support.DefaultPluginCollection.add(DefaultPluginCollection.java:62)
        at modelengine.fitframework.runtime.support.AbstractRootPlugin.loadPlugin(AbstractRootPlugin.java:206)
        at modelengine.fit.plugin.dynamic.directory.DynamicPluginScanner.onFileCreated(DynamicPluginScanner.java:145)
        at modelengine.fitframework.filesystem.support.RegularFileVisitor.visitFile(RegularFileVisitor.java:74)
        at modelengine.fitframework.filesystem.support.RegularFileVisitor.visitFile(RegularFileVisitor.java:37)
        at java.base/java.nio.file.Files.walkFileTree(Files.java:2811)
        at java.base/java.nio.file.Files.walkFileTree(Files.java:2882)
        at modelengine.fitframework.filesystem.support.DefaultDirectoryMonitor.refresh(DefaultDirectoryMonitor.java:138)
        at modelengine.fitframework.schedule.support.DefaultTask$Builder.lambda$runnable$0(DefaultTask.java:93)
        at modelengine.fitframework.schedule.support.DefaultTask.call(DefaultTask.java:37)
        at modelengine.fitframework.schedule.support.ReschedulableTask.call(ReschedulableTask.java:104)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-03 22:55:24.952] [INFO ] [netty-http-server-thread-0] [modelengine.fit.http.server.netty.NettyHttpClassicServer] Start netty http server successfully. [httpPort=8080]
[2025-08-03 22:55:25.330] [INFO ] [directory-monitor-fit-framework-thread-0] [modelengine.fit.plugin.dynamic.directory.DynamicPluginScanner] Dynamic plugin detected. [plugin=E:\home\fred\git\开源框架\fit-framework\build\lib\fit-aop-3.5.0-M6.jar]
[2025-08-03 22:55:25.351] [ERROR] [directory-monitor-fit-framework-thread-0] [modelengine.fit.plugin.dynamic.directory.DynamicPluginScanner] Failed to scan directory.
java.lang.IllegalStateException: Missing manifest in plugin JAR. [location=file:/C:/Users/geex38/AppData/Local/Temp/application-5997341732875804306/fit-aop-3.5.0-M6-17524366135665660030.jar, entry=FIT-INF/plugin.xml]
        at modelengine.fitframework.plugin.support.JarPluginResolver.readPluginManifest(JarPluginResolver.java:115)
        at modelengine.fitframework.plugin.support.JarPluginResolver.loadMetadata(JarPluginResolver.java:63)
        at modelengine.fitframework.plugin.support.JarPluginResolver.resolve(JarPluginResolver.java:57)
        at modelengine.fitframework.plugin.support.DefaultPluginCollection.lambda$add$0(DefaultPluginCollection.java:69)
        at modelengine.fitframework.util.LockUtils.synchronize(LockUtils.java:138)
        at modelengine.fitframework.plugin.support.DefaultPluginCollection.add(DefaultPluginCollection.java:62)
        at modelengine.fitframework.runtime.support.AbstractRootPlugin.loadPlugin(AbstractRootPlugin.java:206)
        at modelengine.fit.plugin.dynamic.directory.DynamicPluginScanner.onFileCreated(DynamicPluginScanner.java:145)
        at modelengine.fitframework.filesystem.support.RegularFileVisitor.visitFile(RegularFileVisitor.java:74)
        at modelengine.fitframework.filesystem.support.RegularFileVisitor.visitFile(RegularFileVisitor.java:37)
        at java.base/java.nio.file.Files.walkFileTree(Files.java:2811)
        at java.base/java.nio.file.Files.walkFileTree(Files.java:2882)
        at modelengine.fitframework.filesystem.support.DefaultDirectoryMonitor.refresh(DefaultDirectoryMonitor.java:138)
        at modelengine.fitframework.schedule.support.DefaultTask$Builder.lambda$runnable$0(DefaultTask.java:93)
        at modelengine.fitframework.schedule.support.DefaultTask.call(DefaultTask.java:37)
        at modelengine.fitframework.schedule.support.ReschedulableTask.call(ReschedulableTask.java:104)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-03 22:55:26.379] [INFO ] [directory-monitor-fit-framework-thread-0] [modelengine.fit.plugin.dynamic.directory.DynamicPluginScanner] Dynamic plugin detected. [plugin=E:\home\fred\git\开源框架\fit-framework\build\lib\fit-aop-aspect-3.5.0-M6.jar]
[2025-08-03 22:55:26.397] [ERROR] [directory-monitor-fit-framework-thread-0] [modelengine.fit.plugin.dynamic.directory.DynamicPluginScanner] Failed to scan directory.
java.lang.IllegalStateException: Missing manifest in plugin JAR. [location=file:/C:/Users/geex38/AppData/Local/Temp/application-5997341732875804306/fit-aop-aspect-3.5.0-M6-12587933242230901754.jar, entry=FIT-INF/plugin.xml]
        at modelengine.fitframework.plugin.support.JarPluginResolver.readPluginManifest(JarPluginResolver.java:115)
        at modelengine.fitframework.plugin.support.JarPluginResolver.loadMetadata(JarPluginResolver.java:63)
        at modelengine.fitframework.plugin.support.JarPluginResolver.resolve(JarPluginResolver.java:57)
        at modelengine.fitframework.plugin.support.DefaultPluginCollection.lambda$add$0(DefaultPluginCollection.java:69)
        at modelengine.fitframework.util.LockUtils.synchronize(LockUtils.java:138)
        at modelengine.fitframework.plugin.support.DefaultPluginCollection.add(DefaultPluginCollection.java:62)
        at modelengine.fitframework.runtime.support.AbstractRootPlugin.loadPlugin(AbstractRootPlugin.java:206)
        at modelengine.fit.plugin.dynamic.directory.DynamicPluginScanner.onFileCreated(DynamicPluginScanner.java:145)
        at modelengine.fitframework.filesystem.support.RegularFileVisitor.visitFile(RegularFileVisitor.java:74)
        at modelengine.fitframework.filesystem.support.RegularFileVisitor.visitFile(RegularFileVisitor.java:37)
        at java.base/java.nio.file.Files.walkFileTree(Files.java:2811)
        at java.base/java.nio.file.Files.walkFileTree(Files.java:2882)
        at modelengine.fitframework.filesystem.support.DefaultDirectoryMonitor.refresh(DefaultDirectoryMonitor.java:138)
        at modelengine.fitframework.schedule.support.DefaultTask$Builder.lambda$runnable$0(DefaultTask.java:93)
        at modelengine.fitframework.schedule.support.DefaultTask.call(DefaultTask.java:37)
        at modelengine.fitframework.schedule.support.ReschedulableTask.call(ReschedulableTask.java:104)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:840)
[2025-08-03 22:55:27.415] [INFO ] [directory-monitor-fit-framework-thread-0] [modelengine.fit.plugin.dynamic.directory.DynamicPluginScanner] Dynamic plugin detected. [plugin=E:\home\fred\git\开源框架\fit-framework\build\lib\fit-aop-bytebuddy-3.5.0-M6.jar]
[2025-08-03 22:55:27.423] [ERROR] [directory-monitor-fit-framework-thread-0] [modelengine.fit.plugin.dynamic.directory.DynamicPluginScanner] Failed to scan directory.
java.lang.IllegalStateException: Missing manifest in plugin JAR. [location=file:/C:/Users/geex38/AppData/Local/Temp/application-5997341732875804306/fit-aop-bytebuddy-3.5.0-M6-12589976988095720703.jar, entry=FIT-INF/plugin.xml]
        at modelengine.fitframework.plugin.support.JarPluginResolver.readPluginManifest(JarPluginResolver.java:115)
        at modelengine.fitframework.plugin.support.JarPluginResolver.loadMetadata(JarPluginResolver.java:63)
        at modelengine.fitframework.plugin.support.JarPluginResolver.resolve(JarPluginResolver.java:57)
        at modelengine.fitframework.plugin.support.DefaultPluginCollection.lambda$add$0(DefaultPluginCollection.java:69)
        at modelengine.fitframework.util.LockUtils.synchronize(LockUtils.java:138)
        at modelengine.fitframework.plugin.support.DefaultPluginCollection.add(DefaultPluginCollection.java:62)
        at modelengine.fitframework.runtime.support.AbstractRootPlugin.loadPlugin(AbstractRootPlugin.java:206)
        at modelengine.fit.plugin.dynamic.directory.DynamicPluginScanner.onFileCreated(DynamicPluginScanner.java:145)
        at modelengine.fitframework.filesystem.support.RegularFileVisitor.visitFile(RegularFileVisitor.java:74)
        at modelengine.fitframework.filesystem.support.RegularFileVisitor.visitFile(RegularFileVisitor.java:37)
        at java.base/java.nio.file.Files.walkFileTree(Files.java:2811)
        at java.base/java.nio.file.Files.walkFileTree(Files.java:2882)
        at modelengine.fitframework.filesystem.support.DefaultDirectoryMonitor.refresh(DefaultDirectoryMonitor.java:138)
        at modelengine.fitframework.schedule.support.DefaultTask$Builder.lambda$runnable$0(DefaultTask.java:93)
        at modelengine.fitframework.schedule.support.DefaultTask.call(DefaultTask.java:37)
        at modelengine.fitframework.schedule.support.ReschedulableTask.call(ReschedulableTask.java:104)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:840)
```

### 额外信息 / Additional Context

v3.5.0-M6

### 确认事项 / Confirmations

- [x] 我已经搜索了现有的issues，确认这不是重复问题
I have searched existing issues and confirmed this is not a duplicate

- [x] 我已经阅读了项目文档和 FAQ
I have read the project documentation and FAQ  

- [x] 我提供的信息准确完整
The information I provided is accurate and complete

- [x] 我愿意协助测试问题修复 (可选)
I'm willing to help test the bug fix (optional)

--- a/README.md
+++ b/README.md
@@ -106,7 +106,9 @@ chmod +x build/bin/*
 **启动命令**
 
 ```
-build/bin/fit start
+mkdir dynamic-plugins
+cd dynamic-plugins
+../build/bin/fit start
 ```
 
 > 以上编译构建出的 `fit` 命令可以通过系统操作（别名或添加系统路径）来简化输入。
@@ -119,6 +121,8 @@ build/bin/fit start
 - 新建任意目录作为插件目录，在该目录下存放插件，可在插件目录下使用命令 `fit start` 启动服务。
 
 > 以上环境配置步骤请根据使用的操作系统使用相应的路径分隔符和环境变量配置操作。
+> 
+> `fit` 命令无法在包含 FIT 框架的目录中执行，只能够在一个新建的动态插件目录下执行。
 
 ## 快速开始
 

